File: python/bc1555.py
from math import pow

res = ""

count = int(input())

for c in range(0, count):
    cVal = input().split()

    x = int(cVal[0])
    y = int(cVal[1])

    ra = pow(3*x, 2) + pow(y, 2);
    be = (2 * pow(x, 2)) +  pow(5*y, 2);
    ca = -100*x + pow(y, 3);

    if ra > be and ra > ca: res = "Rafael ganhou"
    elif be > ra and be > ca: res = "Beto ganhou"
    else: res = "Carlos ganhou"

    print(res)

File: cpp/bc1555.cpp
#include <iostream>
#include <cmath>

using namespace std;

int main()
{
    int x, y, count;
    double ra, be, ca;
    string res = "";

    cin >> count;
    
    for(int i = 0; i  < count; i++)
    {
        cin >> x >> y;

        ra = pow(3*x, 2) + pow(y, 2);
        be = (2 * pow(x, 2)) +  pow(5*y, 2);
        ca = -100*x + pow(y, 3);

        if(ra > be && ra > ca) res = "Rafael ganhou";
        else if(be > ra && be > ca) res = "Beto ganhou";
        else res = "Carlos ganhou";

        cout << res << endl;
    }
}